k8s/spicedb.yaml

```diff
@@ -26,56 +33,116 @@ spec:
       port: 50051
       protocol: "TCP"
       targetPort: 50051
-    - name: "internal"
+    - name: "http"
+      port: 8443
+      protocol: "TCP"
+      targetPort: 8443
+    - name: "dispatch"
       port: 50053
       protocol: "TCP"
       targetPort: 50053
+    - name: "prometheus"
+      port: 9090
+      protocol: "TCP"
+      targetPort: 9090
 ---
 apiVersion: "apps/v1"
 kind: "Deployment"
 metadata:
   name: "spicedb"
+  labels:
+    app.kubernetes.io/name: "spicedb"
+    app.kubernetes.io/version: "example"
```

Comment:
Instead of "example" we can say 1.0.0

Reply:
What is the purpose of this label? Is it used any where?

Reply:
These labels were simply taken from a template on a spicedb repo, totally open to changing all of this stuff. 

As said in my comment, above, I wonder if it is worth trying to run the prod DeploymentConfigs locally (with some changes).

Reply:
FYI, these are recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
"These are recommended labels. They make it easier to manage applications but aren't required for any core tooling."

The version one should match the version of spicedb, but I think there no point in maintaining a version here. I'll change it to "latest". (If you really want I can just delete them. :) )